src/hoarder/archives/rar_archive_7z.py

```
@@ -0,0 +1,214 @@
+        command_line = [
+            str(SEVENZIP),
+            "l",
+            "-slt",
+            "-scsUTF-8",
+            "-sccUTF-8",
+            "-p" + (password if password else ""),
+            str(path),
+        ]
+
+        try:
+            sub = subprocess.run(command_line, capture_output=True, check=True)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)